collections/_priorities/platforms-03-improve-gamepad-support.md

```diff
@@ -0,0 +1,19 @@
+---
+last_updated: "2024-11-30"
+category: platforms
+rank: 3
+created: "2024-11-30"
+stage:
+  # Must be dates (YYYY-MM-DD), empty if not there yet.
+  planning: ""
+  ongoing: ""
+  done: ""
+anchor: "improve-gamepad-support"
+title: "Improve gamepad support"
+description: |
+  We often receive issue reports about gamepad support, as the numbers of gamepads never cease to increase. As we wish to support as many input devices as possible, we want to work towards better support.
+details:
+  - type: note
+    content: |
+      We need to investigate how we can do this. One way would be to incorporate more code from SDL, or to integrate SDLInput directly in the engine.
```

Comment:
The canonical name for SDL's libraries are "SDL_image", "SDL_mixer", "SDL_input" and so on.

```suggestion
      We need to investigate how we can do this. One way would be to incorporate more code from SDL, or to integrate SDL_input directly in the engine.
```